src/Craftwork/IdObfuscator/IdObfuscator.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace Craftwork\IdObfuscator;
+
+final class IdObfuscator
+{
+    /**
+     * @var CharacterSet
+     */
+    private $characterSet;
+
+    /**
+     * @param string       $salt Salt that should be unique and kept secret for your application.
+     * @param CharacterSet $characterSet An optional character set.
+     */
+    public function __construct(string $salt, CharacterSet $characterSet = null)
+    {
+        $this->characterSet = (null === $characterSet ? CharacterSet::ofDefaultCharacters() : $characterSet)
```

Comment:
What do you think about using the null-coalescing operator e.g. `$this->characterSet = $characterSet ?? CharacterSet::ofDefaultCharacters()`?